src/Controller/ROIOptionsController.py

```diff
@@ -150,3 +151,40 @@ def show_roi_manipulate_options(self, roi_color):
         else:
             self.manipulate_window.update_ui(rois, dataset_rtss, roi_color)
         self.manipulate_window.show()
+
+
+class ROITransferOptionUI(QtWidgets.QMainWindow, UITransferROIWindow):
```

Comment:
Please add docstrings to these two classes or their `__init__` functions.

Reply:
done